source/extensions/filters/http/ext_authz/ext_authz.cc

```diff
@@ -58,6 +59,9 @@ void Filter::initiateCall(const Http::RequestHeaderMap& headers,
       config_->includePeerCertificate(), config_->destinationLabels());
 
   ENVOY_STREAM_LOG(trace, "ext_authz filter calling authorization server", *decoder_callbacks_);
+  // Store start time of ext_authz filter call
+  start_time_ = decoder_callbacks_->dispatcher().timeSource().monotonicTime();
```

Comment:
Seems like this can be a local variable (vs. class member)?

Reply:
It can't be a local variable without rewriting what I'm doing since I explicitly reference that variable as the start time in the on complete method.